src/components/views/elements/ImageView.tsx

```diff
@@ -44,6 +45,8 @@ const ZOOM_COEFFICIENT = 0.0025;
 // If we have moved only this much we can zoom
 const ZOOM_DISTANCE = 10;
 
+const PANEL_HEIGHT = 68;
```

Comment:
Where is this height coming from? I assume it's from `res/css/views/elements/_ImageView.scss:43`, could we use css variable for that instead, so that we have one entry point for a magic number? 😄

Reply:
I added a comment as I didn't understand what you meant at first :sweat_smile: Is that satisfactory or do you think having a SCSS variable would be better?

Reply:
https://davidwalsh.name/css-variables-javascript have a look here, it'd make sense to move the `height: 68px` from `res/css/views/elements/_ImageView.scss:43` to a css variable and then refer here to that css variable instead, so that we don't have to update it in two places in the future, preventing ourselves from silly regressions. 😄

Reply:
Ah, didn't realize we can do that, thanks